cli/src/cmd/forge/doc.rs

```diff
@@ -39,6 +39,10 @@ pub struct DocArgs {
     /// Port for serving documentation.
     #[clap(long, short, requires = "serve")]
     port: Option<usize>,
+
+    /// Option to generate documentation for deployments via `hardhat-deploy` / `forge-deploy`.
+    #[clap(long, short)]
+    deployments: bool,
```

Comment:
Yeah, definitely does. Added this in, it will only accept a relative path from the project root at the moment. Should we also support absolute paths? Most `hardhat-deploy` / `forge-deploy` projects store the deployment artifacts within the project, so I don't think this is a big concern.

Reply:
> store the deployment artifacts within the project,

makes sense, 

`.join` absolute paths will use the absolute path so I think this should still work, but since this is very niche, can ignore this case.